assets/semgrep_rules/client/privacy.yaml

```diff
@@ -0,0 +1,30 @@
+rules:
+  - id: privacy
+    metadata:
+      author: Andrea Brancaleoni <abc@pompel.me>
+      confidence: LOW
+      assignees: |
+        bridiver
+        arthuredelstein
+        ShivanKaul
+        aaronsantamaria
```

Comment:
I'm guessing we don't want to ping aaron for this, though I'm sure his inbox filter-foo is pretty advanced

Reply:
I got the `privacy-text` users. I can remove him though